src/main/scala/gitbucket/core/service/RepositoryService.scala

```diff
@@ -434,7 +434,7 @@ trait RepositoryService { self: AccountService =>
     val preferred = PluginRegistry().renderableExtensions.map(extension => s"${fileBaseName}.${extension}")
 
     def choiceTemplate(files: List[FileInfo]): Option[FileInfo] =
-      files.find(f => preferred.contains(f.name)).orElse(files.find(f => f.name.startsWith(fileBaseName)))
+      files.find(f => preferred.contains(f.name)).orElse(files.find(f => fileBaseName == FileUtil.trimExtension(f.name)))
```

Comment:
According to [this](https://help.github.com/articles/creating-an-issue-template-for-your-repository/),  template files are not case sensitive.
And files without extensions seem to have higher priority than files with extensions(as @McFoggy said).
So, I'll fix as above. Please just a moment.